samples/app/src/main/java/com/sourcepoint/app/v6/web/WebConsentTransferTestActivity.kt

```diff
@@ -0,0 +1,137 @@
+package com.sourcepoint.app.v6.web
+
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.webkit.WebChromeClient
+import android.webkit.WebViewClient
+import androidx.appcompat.app.AppCompatActivity
+import com.sourcepoint.app.v6.R
+import com.sourcepoint.cmplibrary.NativeMessageController
+import com.sourcepoint.cmplibrary.SpClient
+import com.sourcepoint.cmplibrary.core.nativemessage.MessageStructure
+import com.sourcepoint.cmplibrary.creation.delegate.spConsentLibLazy
+import com.sourcepoint.cmplibrary.exception.CampaignType
+import com.sourcepoint.cmplibrary.model.ConsentAction
+import com.sourcepoint.cmplibrary.model.MessageLanguage
+import com.sourcepoint.cmplibrary.model.exposed.SPConsents
+import com.sourcepoint.cmplibrary.util.clearAllData
+import com.sourcepoint.cmplibrary.util.extensions.preloadConsent
+import kotlinx.android.synthetic.main.activity_web_consent_transfer_test.*
+import org.json.JSONObject
+
+class WebConsentTransferTestActivity : AppCompatActivity() {
+
+    private val consentWebViewClient = object : WebViewClient() { }
+    private val consentWebChromeClient = object : WebChromeClient() { }
+
+    private val spConsentLib by spConsentLibLazy {
+        activity = this@WebConsentTransferTestActivity
+        spClient = LocalClient()
+        config {
+            accountId = 22
+            propertyId = 16893
+            propertyName = "mobile.multicampaign.demo"
```

Comment:
@bohdan-go-wombat have you tested it with any other property?

Reply:
no, I needed a property with both `CCPA` and `GDPR` campaign, I decided the one that I'm 100% sure, and also, the main goal of those tests are to test web consent transfer flow, so if it passes on one property - it should on any others as well

do you think that is crucial to test on another properties as well?

Reply:
Testing a single property is enough for this feature.